<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <!-- <arg name="rviz" default="false"/> -->

    <!-- Interpolator -->
    <node name="interpolator" pkg="tracking_pid" type="path_interpolator">
        <param name="target_x_vel" value="0.5"/>
        <param name="target_yaw_vel" value = "0.5"/>
        <param name="target_yaw_acc" value = "0.2"/>
        <param name="target_x_acc" value="100"/>
    </node>

    <!-- PID controller -->
    <node name="controller" pkg="tracking_pid" type="controller" output="screen">
      <!-- <remap from="/path" to="/irl_path"/> -->
      <remap from="move_base/cmd_vel" to="/cmd_vel"/>
      <remap from="local_trajectory" to="trajectory"/>
      <param name="controller_debug_enabled" value="True"/>
      <param name="l" value="0.2"/>
    </node>

</launch>